pandas/core/base.py

```diff
@@ -950,10 +958,32 @@ def argmax(self, axis=None, skipna=True, *args, **kwargs):
         The maximum cereal calories is the third element and
         the minimum cereal calories is the first element,
         since series is zero-indexed.
+
+        >>> s2 = pd.Series({{'A': 100.0, 'B': 110.0,
+        ...                'C': 120.0, 'D': 110.0,
+        ...                'E': 120.0, 'F': 100.0}})
+
+        >>> s2.argmax(keep='all')
+        array([2, 4])
+        >>> s2.argmin(keep='all')
+        array([0, 5])
+
+        >>> s2.argmax(keep='last')
+        4
+        >>> s2.argmin(keep='last')
+        5
         """
         nv.validate_minmax_axis(axis)
         nv.validate_argmax_with_skipna(skipna, args, kwargs)
-        return nanops.nanargmax(self._values, skipna=skipna)
+        if keep == "last":
+            return (self.size - 1) - nanops.nanargmax(self._values[::-1], skipna=skipna)
+        elif keep == "all":
+            temp = self.reset_index(drop=True)
+            return temp[
+                temp == temp[nanops.nanargmax(temp._values, skipna=skipna)]
+            ].index.values
+        else:
```

Comment:
this should be elif keep == "first" and else should be a raise with an invalid keep parameter